nemo/collections/asr/parts/submodules/multi_head_attention.py

```diff
@@ -350,8 +349,9 @@ def forward(self, x, cache_len=0):
         # negative positions would be used for right and positive for left tokens
         # for input of length L, 2*L-1 positions are needed, positions from (L-1) to -(L-1)
         input_len = x.size(1) + cache_len
-        start_pos = self.center_pos - input_len
-        end_pos = self.center_pos + input_len - 1
+        center_pos = self.pe.size(1) // 2 + 1
```

Comment:
Was center_pos causing any issue?

Reply:
How about registering it as a buffer ?

Reply:
The problem was that this constant should not have been a tensor - and being a tensor of int32, it also generated ONNXRT error that wanted all slice args be same int type (and used int64 for other calculated expr). 
registering it as a buffer does not make any sense - it's a scalar and should not be flipped to and from GPU, also it's fine for ONNX exporter to treat it as constant. If we want to have dynamic resize in ONNX - my fix is already fine at low cost of 2 scalar ops - but it would also require more changes on resize side.